frontend/model/contracts/chatroom.js

```diff
@@ -318,8 +415,8 @@ sbp('chelonia/defineContract', {
         }
       },
       sideEffect ({ meta, contractID }, { state }) {
-        if (!state.saveMessage && state.attributes.creator === meta.username) { // Not sure this condition is necessary
-          if (sbp('okTurtles.data/get', 'JOINING_CHATROOM')) {
+        if (state.attributes.creator === meta.username) { // Not sure this condition is necessary
```

Comment:
Is this `// Not sure this condition is necessary` still true?

Reply:
It's not necessary. We should remove that condition.
All the members should leave chatroom if the chatroom is deleted.